package.json

```diff
@@ -40,14 +39,15 @@
     "ember-export-application-global": "^1.0.5",
     "ember-frost-core": "^1.14.3",
     "ember-frost-test": "^1.0.0",
-    "ember-load-initializers": "^0.5.1",
-    "ember-prop-types": "3.11.0",
+    "ember-load-initializers": "^0.6.0",
+    "ember-prop-types": "^3.11.0",
```

Comment:
This is not the same value that was set in the blueprint.  The blueprint is `3.11.0`, which is the original value of this diff.